macos/snd_.ksy

```diff
@@ -0,0 +1,201 @@
+meta:
+  id: snd_
```

Comment:
At the moment, KSY spec identifiers aren't namespaced, so all spec identifiers in kaitai_struct_formats should be descriptive on their own, without the directory structure. A name like `mac_os_snd` or `mac_os_sound_resource` would be better.

Also, I don't know if/how well the compiler supports trailing underscores. It's not obvious how something like `snd_` should be convered to CamelCase (`Snd` or `Snd_`?). Just to be safe, I would choose an identifier that doesn't end with an underscore.

Reply:
changed to mac_os_resource_snd